### DISCUSSION TITLE
Why Canvas cannot show highlight in PDF？

### DISCUSSION THREAD

**Discussion by alexweyy** [RyotaUshio/obsidian-pdf-plus#229] [Q&A]
When I reference text in pdf to canvas, 
the referenced text in PDF cannot be highlighted.
The markdown file does not have this problem.

Still, this is a great plugin

![recording](https://github.com/RyotaUshio/obsidian-pdf-plus/assets/133588166/6241e0a3-d721-4394-8290-da0132551ecc)

**Comment by RyotaUshio** [comment]
Thanks!
Linking from within Canvas is not supported yet. This is a limitation coming from the Obsidian core.
Please refer to #90 & #43.

**Reply by alexweyy** [reply]
Thanks reply, I think this plugin is the greatest project！
